ext/ffi/lib.rs

```diff
@@ -2065,7 +2065,7 @@ fn op_ffi_call_nonblocking<'scope>(
 #[op(fast)]
 fn op_ffi_ptr_of<FP>(
   state: &mut deno_core::OpState,
-  buf: &[u8],
+  buf: *const u8,
```

Comment:
Consider using `*mut u8`, not because we're going to do mutation  but because we want this to be from an ArrayBuffer which are always mut.

Reply:
`*const u8` does not suggest that the pointed value is _not_ mutable. You can safely cast it to `*mut u8`. It's just a hint that we are only dealing with the non-mut reference in the current context.

Reply:
Yes, I'm just thinking that there might be a day when `*const u8` is needed to stand in for some V8 API type with Fast API support that offers a pointer without mutable access. Admittedly this is quite an unlikely thing scenario, but still a theoretical possibility.

I guess another way to go about this would use some transparent newtype like to tell the various pointer types apart in the ops macro:
```rs
#[repr(transparent)]
struct Uint8ArrayPointer(*const u8)
```

But yeah, since this is an unlikely scenario it doesn't really matter.